src/scss/components/switch/_index.scss

```diff
@@ -51,6 +53,7 @@ hx-switch-control > input[type="checkbox"] ~ label {
     text-transform: uppercase;
     margin-left: 2rem;
     color: $cyan-900;
+    color: var(--hxSwitchControl-switch-offlabel-color, $cyan-900);
```

Comment:
Please use updated colors to replace `$cyan-900`.

Reply:
Changes have been done, please review.